### ISSUE TITLE
RFC: Synchronization problems in 0.63

### ISSUE THREAD

**Issue by zadam** [TriliumNext/Trilium#4613]
### Describe feature

Hello, since 0.61 there has been a lot of trouble / bugs with the synchronization. I fixed all the problems I could find / identify and it looks like it did help for some people, but others still seem to be having problems.

I'd like to get this working for everybody, but for that I need help from those who encounter problems. Here's what I need to diagnose / debug the problem:

*   I need an anonymized database from **both client and server**. You can anonymize the database in Options -> Advanced.  You can find them in the `anonymized-db` subdirectory of the "data directory" which can be seen in the "About Trilium Notes".
*   I need both client and server logs. You can find the `log` subdirectory of the "data directory" which can be seen in the "About Trilium Notes". Please include the logs for the whole days, not just excerpts showing the errors.

ZIP these files and send them to zadam.apps@gmail.com. Thank you!

**Comment by rom1dep** [comment]
@zadam is this what's holding back 0.63?

**Comment by zadam** [comment]
Yes, I'd like to get the sync finally sorted out. The other thing is that 0.63 contains only minor improvements, so there's not much pressure to release.

**Comment by justyns** [comment]
FWIW I was having a lot of sync issues after upgrading, however I haven't experienced any in the recent weeks.  I think what finally worked for me was making a backup of my trilium db, deleting the db files from all of my clients, importing the db into a single client and then start syncing from scratch.

**Comment by perfectra1n** [comment]
Sent an email, that includes the information you requested above! Hopefully it helps :)

**Comment by eliandoran** [comment]
The sync issues should have been sorted out by now.
